hbase-client/src/main/java/org/apache/hadoop/hbase/ipc/NettyRpcConnection.java

```diff
@@ -347,7 +348,7 @@ public void operationComplete(ChannelFuture future) throws Exception {
   private void sendRequest0(Call call, HBaseRpcController hrc) throws IOException {
     assert eventLoop.inEventLoop();
     if (reloginInProgress) {
-      throw new IOException("Can not send request because relogin is in progress.");
+      throw new IOException(HConstants.RELOGIN_IS_IN_PROGRESS);
```

Comment:
This is weird, please don't put these kinds of constants in HConstants. There are too many unrelated concerns there already. 

Public static string constant in some other file, even this one, is preferred.

Reply:
initially wanted to keep in this class but since it belongs to hbase-client, hbase-server RSProcedureDispatcher can't access it

Reply:
not sure what is the best place to keep this, anywhere else in hbase-common would also work

Reply:
Please do not put it in HConstants, it is IA.Public.

Reply:
i understand but i am not sure what is the best place to keep this in

Reply:
ok, this is now taken care of